details.html

```diff
@@ -577,9 +581,348 @@
+                          <span class="material-icons-round">
+                            delete_outline
+                          </span>
+                        </button>
+                      </div>
+                    </a>
+                  </li>
+
+                  <li>
+                    <div role="button" href="#" class="task">
+                      <span class="task-name">Testing for possible errors</span>
+
+                      <div class="actions">
+                        <button class="btn small transparent icon-only">
+                          <span class="material-icons-round">
+                            drag_indicator
+                          </span>
+                        </button>
+
+                        <button class="btn small transparent icon-only">
+                          <span class="material-icons-round">
+                            delete_outline
+                          </span>
+                        </button>
+                      </div>
+                    </a>
+                  </li>
+
+                  <li>
+                    <div role="button" href="#" class="task">
+                      <span class="task-name">
+                        Final debugging applications
+                      </span>
+
+                      <div class="actions">
+                        <button class="btn small transparent icon-only">
+                          <span class="material-icons-round">
+                            drag_indicator
+                          </span>
+                        </button>
+
+                        <button class="btn small transparent icon-only">
+                          <span class="material-icons-round">
+                            delete_outline
+                          </span>
+                        </button>
+                      </div>
+                    </a>
+                  </li>
+
+                  <li>
+                    <button class="btn link primary add-task">
+                      <span class="material-icons-round">add</span>
+                      Add Checklist Item
+                    </button>
+                  </li>
+                </ul>
+              </section>
+            </article>
+
+            <section class="activity">
+              <nav class="tabs list-unstyled">
+                <li>
+                  <button class=" tab content-title active">comments</button>
+                </li>
+
+                <li>
+                  <button class=" tab content-title">activity</button>
+                </li>
+              </nav>
+
+              <div class="comment-section">
+                <textarea
+                  cols="30"
+                  rows="3"
+                  placeholder="Add Comment..."
+                ></textarea>
+
+                <div class="footer">
+                  <button class="btn small primary">Comment</button>
+
+                  <div class="actions">
```

Comment:
Wrong class name. "Comment" button is also an action, but it appears outside of your `.actions`. Name it correctly. These three buttons do the same action: they attach something.